bleak/backends/corebluetooth/client.py

```diff
@@ -361,7 +361,10 @@ async def start_notify(
         """
         assert self._delegate is not None
 
-        await self._delegate.start_notifications(characteristic.obj, callback)
+        cb = kwargs.get("cb") or dict()
```

Comment:
Would be better to make this an explicit keyword-only argument, then we can have type hints for it and we can document it in the "frontend" `start_notify()` function in `__init__.py`.

Reply:
I made it an explicit argument in `corebluetooth/client.py`.  I didn't yet expose it explicitly in` __init__.py`, but let me know if you want me to go that far.

Reply:
Yes, that is what I was suggesting.

Reply:
I started going down this path, and ran right into the `TypedDict` comment below, and was unsure what to name this. `CBStartNotifyArgs` maybe?

If you'd like this exposed at the `__init__.py` level, just let me know what you'd like the Args dict to be named and where it should live.